cmd/webhook/main.go

```diff
@@ -213,6 +215,22 @@ func main() {
 		SecretName:  secretName,
 	})
 
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/", handler)
+	mux.HandleFunc("/health", healthHandler)
+	mux.HandleFunc("/readiness", readinessHandler)
+
+	// use PORT environment variable, or default to 8080
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+	}
+
+	// start the web server on port and accept requests
+	log.Printf("Server listening on port %s", port)
+	log.Fatal(http.ListenAndServe(":"+port, mux))
```

Comment:
I think this will block and `sharedmain.WebhookMainWithConfig` won't run until `ListenAndServe` returns.  Though I might be missing something?

Reply:
Ah oops forgot to do it concurrently.